qt/editor_dialog.cpp

```diff
@@ -81,15 +81,15 @@ EditorDialog::EditorDialog(QWidget * parent, osm::EditableMapObject & emo)
 
   if (emo.IsAddressEditable())
   {  // Address rows.
-    vector<string> nearbyStreets = emo.GetNearbyStreets();
+    vector<osm::EditableMapObject::TLocalizedStreet> nearbyStreets = emo.GetNearbyStreets();
     // If feature does not have a specified street, display empty combo box.
-    if (emo.GetStreet().empty())
-      nearbyStreets.insert(nearbyStreets.begin(), "");
+    if (emo.GetStreet().first.empty())
+      nearbyStreets.insert(nearbyStreets.begin(), {"", ""});
```

Comment:
Пустую пару :)

> On 8.4.2016, at 17.40, Vlad Mihaylenko <notifications@github.com> wrote:
> 
> In qt/editor_dialog.cpp:
> 
> >      // If feature does not have a specified street, display empty combo box.
> > -    if (emo.GetStreet().empty())
> > -      nearbyStreets.insert(nearbyStreets.begin(), "");
> > +    if (emo.GetStreet().first.empty())
> > +      nearbyStreets.insert(nearbyStreets.begin(), {"", ""});
> ну, вроде логично, мы же пару инсертим.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>